nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -172,6 +172,8 @@
 
 - [Fedimint](https://github.com/fedimint/fedimint), a module based system for building federated applications (Federated E-Cash Mint). Available as [services.fedimintd](#opt-services.fedimintd).
 
+- [Ryzen-ppd](https://github.com/xsmile/ryzen-ppd), a power profiles daemon driver for AMD Ryzen laptops. Available as programs.ryzen-ppd.
```

Comment:
If anyone knows how I can link the option reference in the end of the sentence to something like `#opt-programs.ryzen-ppd` please let me know. I can't get the manual to build with that link in place and I dunno how to make that reference be valid.

Reply:
@NixOS/documentation-team Please can one of you help me out :)